src/main/resources/static/static/js/updateOrder.js

```diff
@@ -0,0 +1,68 @@
+//计算商品总价
+function updateFinalPrice() {
+    var $tr = $("#itemTable").find("tr[id]");
+    var summer = 0;
+    $tr.each(function(i, dom) {
+        if($(dom).find("#sItemChecked").is(":checked")){
+            var num = $(dom).find(".ItemNo").val(); //商品数量
+            var name = $(dom).find("#ItemName").text();//商品名称
```

Comment:
Can you make sure if there're multiple items in the cart, will the name and the number matching one by one? 

For example, will it working as:

show  itemName==2 but itemNo == 1 ?

Reply:
after my test,it would be matching one by one

Reply:
Thanks.